README.md

```diff
@@ -19,7 +19,9 @@ TiDB vector supports below distance functions:
 - `CosineDistance`
 - `NegativeInnerProduct`
 
-supports following orm or framework:
+It also supports using hnsw index with l2 or cosine distance to speed up the search, for more details see [Vector Search Indexes in TiDB](https://docs.google.com/document/d/15eAO0xrvEd6_tTxW_zEko4CECwnnSwQg8GGrqK1Caiw)
```

Comment:
We need to replace this google doc by a more formal web doc

Reply:
After the public preview, we will create a more formal web document on docs.pingcap.com, and I will update it accordingly.